k9mail-library/src/test/java/com/fsck/k9/mail/filter/SmtpDataStuffingTest.java

```diff
@@ -0,0 +1,79 @@
+package com.fsck.k9.mail.filter;
+
+import java.io.IOException;
+import java.io.ByteArrayOutputStream;
+
+import okio.ByteString;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class SmtpDataStuffingTest {
+    private String expectedMessageAfterStuffing;
+    private String actualMessageAfterStuffing;
+
+    @Before
+    public void setUp(){
+        expectedMessageAfterStuffing = "";
+        actualMessageAfterStuffing = "";
+    }
+
+    @Test
+    public void smtpDotStuffing_OnlyStuffingDotTest() throws IOException {
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+        SmtpDataStuffing smtpDataStuffingStream = new SmtpDataStuffing(byteArrayOutputStream);
+        byte[] data = ByteString.encodeUtf8("Hello dot\r\n.").toByteArray();
+
+        expectedMessageAfterStuffing = "Hello dot\r\n..";
+        try {
+            smtpDataStuffingStream.write(data);
+        }
+        finally {
+            smtpDataStuffingStream.close();
+            byteArrayOutputStream.close();
+        }
```

Comment:
Sure, I'll do the same! Do you mean a line break after line 36 here (for code style)?